src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -66,15 +71,19 @@ CAC.Map.Control = (function ($, L, _) {
     return MapControl;
 
     function initializeBasemaps() {
-        basemaps.Streets = L.tileLayer('https://stamen-tiles-{s}.a.ssl.fastly.net/toner-lite/{z}/{x}/{y}.png', {
-            attribution: stamenTonerAttribution
-        });
-
         basemaps.Terrain = L.tileLayer('https://stamen-tiles-{s}.a.ssl.fastly.net/terrain/{z}/{x}/{y}.png', {
             attribution: stamenAttribution
         });
 
-        basemaps.Streets.addTo(map);
+        basemaps.Satellite = L.tileLayer('http://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}', {
```

Comment:
I imagine there's an SSL version of this. If so, it would be good to switch this over now to make it easier down the road for serving our app over SSL without a mixed content warning.